### DISCUSSION TITLE
blog/2022-retrospective

### DISCUSSION THREAD

**Discussion by giscus** [theodorusclarence/theodorusclarence.com#337] [General]
# blog/2022-retrospective

Surabaya, Covid, Apple Developer Academy, Projects and more

https://theodorusclarence.com/blog/2022-retrospective

**Comment by theodorusclarence** [comment]
Hello, kindly leave a like (or 5 😄), and comment if you got something to ask! 🚀

**Comment by kissu** [comment]
Congratz for the academy + RIP for COVID. 💀
Overall, glad that you had a nice year! 💪🏻

**Reply by theodorusclarence** [reply]
Thanks! Had a blast!

**Comment by andhikadk** [comment]
tips lolos apple developer academy bang, mau daftar buat tahun depan soalnya😁
